src/components/FlowRunTimeline.vue

```diff
@@ -1,6 +1,34 @@
 <template>
-  <div class="flow-run-timeline">
+  <div
+    class="flow-run-timeline"
+    :class="{ 'flow-run-timeline--fullscreen': isFullscreen }"
+    tabindex="0"
```

Comment:
It is yeah. It also has the benefit of making the graph register as a thing to the screen reader. Otherwise the reader just sees the settings and action buttons in the middle of nowhere.

Reply:
Hm ok, you might need more than `outline-none` on the root component to reset tab styles; I think you'll need something more like:
```css
.flow-run-timeline,
.flow-run-timeline:focus-within,
.flow-run-timeline:focus,
.flow-run-timeline:active {
  outline-none
  border-none
  ring-none
}
```
(haven't vetted this, just a suggestion)

Reply:
Registering the graph for screen reads won't actually allow a keyboard-only user to interact with the graph though, right? Or are individual nodes available for keyboard interaction?

Reply:
Correct. I do have a ticket for adding actual screen reader support down the line, particularly if/when we remove some of the Flow Run Page tabs (e.g. tasks tab) in favor of clicking stuff from the graph. For now it just reads out "Here's a graph" like an image.